torch/csrc/jit/codegen/cuda/compute_at.h

```diff
@@ -13,6 +13,71 @@ namespace fuser {
 
 class TensorView;
 
+// We're going to keep data related to the computeAt pass for each TensorView in
+// this structure, this will allow us to keep a single entry in a map from a
+// TensorView to this one.
+struct ComputeAtData {
```

Comment:
@tlemo recently changed structs to classes except for POD.

Reply:
computeAtData is PPD with some accessors, but he can comment what this should be.

Reply:
since we have non-trivial constructors and public/private, it should be a `class`